### ISSUE TITLE
Add pprof endpoints

### ISSUE THREAD

**Issue by sbueringer** [open-policy-agent/opa#1232]
# Expected Behavior

It would be nice to have the pprof endpoints available (either enabled via a flag or permanently) to make it easier to debug OPA behaviour.

# Actual Behavior

No pprof endpoints are provided.

# Additional Info

I think it should be possible to enable pprof endpoints when necessary via an additional flag. If the flag is set, the pprof handler (https://golang.org/pkg/runtime/pprof/) should be added to the router:
````
router.HandleFunc("/debug/pprof", pprof.Index)
router.Handle("/debug/pprof/heap", pprof.Handler("heap"))
router.Handle("/debug/pprof/allocs", pprof.Handler("allocs"))
router.Handle("/debug/pprof/mutex", pprof.Handler("mutex"))
router.Handle("/debug/pprof/block", pprof.Handler("block"))
router.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
router.HandleFunc("/debug/pprof/profile", pprof.Profile)
router.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
router.HandleFunc("/debug/pprof/trace", pprof.Trace)
````

If it's ok I would open a PR vor this.

**Comment by tsandall** [comment]
@sbueringer a PR would be great. On by default sounds fine to me but if you want plumb a flag through you would modify:

cmd/run.go - add the actual cobra flag
server/server.go - add a function like WithPprofEnabled(bool)
runtime/runtime.go - add field to runtime.Params and call WithPprofEnabled when server is instantiated

**Comment by sbueringer** [comment]
@tsandall I'm just not sure if there are any security considerations if the endpoint is always enabled, that's why I thought a flag might be a good idea.

**Comment by tsandall** [comment]
I think the flag is fairly standard. If you can take that on, that would be great.

**Comment by sbueringer** [comment]
Yes, of course, no problem.

**Comment by tsandall** [comment]
Fixed by #1234
